README.md

```diff
@@ -16,13 +16,14 @@ The SDS Server can protect the private keys via SGX in 2 scenarios: workloads an
 
 Prerequisites for using Istio mTLS private key protection with SGX:
 
-- Kubernetes cluster with one or more nodes with Intel® [SGX](https://software.intel.com/content/www/us/en/develop/topics/software-guard-extensions.html) supported hardware
+- Kubernetes cluster with one or more nodes with [Intel® SGX](https://software.intel.com/content/www/us/en/develop/topics/software-guard-extensions.html) supported hardware
 - [Intel® SGX device plugin for Kubernetes](https://github.com/intel/intel-device-plugins-for-kubernetes/blob/main/cmd/sgx_plugin/README.md)
 - Linux kernel version 5.11 or later on the host (in tree SGX driver)
 - [trusted-certificate-issuer](https://github.com/intel/trusted-certificate-issuer)
```

Comment:
How about to change like this
```suggestion
- [trusted-certificate-issuer](https://github.com/intel/trusted-certificate-issuer) (Optional)
```

Reply:
Okay, fixed. I have mentioned in the `NOTE` below that the `KMRA` and `aesm` service are optional.